feat(payment.html and description.html) add header and footer

Heylin-Loaiza opened the pull request:
Add headers, footers and page name correction

javierbarbozam commented on payment/payment.html:
```diff
@@ -4,9 +4,14 @@
 </head>
 <body>
+  <header>
+    <button>Back</button>
```
Ponete este texto dentro de un span, porque recuerdo que en una de las versiones escondíamos el texto para que solo quedara la flecha

javierbarbozam commented on description/description.html:
```diff
@@ -4,10 +4,18 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>DUFTE</title>
```
Acá podemos poner "DUFTE | Product" o algo así? Para diferenciar la página del index?

javierbarbozam commented on confirmation/confirmation.html:
```diff
@@ -4,7 +4,7 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>DUFTE</title>
```
Acá podemos poner "DUFTE | My cart" o algo así? Para diferenciar la página del index?

javierbarbozam commented on payment/payment.html:
```diff
@@ -4,9 +4,14 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>DUFTE</title>
```
Acá podemos poner "DUFTE | Payment" o algo así? Para diferenciar la página del index?

javierbarbozam reviewed (changes requested):
cositas pequeñas.

Heylin-Loaiza committed:
feat(payment.html and description.html) add header and footer

Add headers, footers and page name correction

Heylin-Loaiza committed:
fix(html) fix header and footer

fix header and footer on confirmation.html, description.html, index.html and payment.html
